.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,52 @@
+name: Bump Version on Release Branch Creation
```

Comment:
Won't this workflow will have unnecessary runs when we will create for example release/1.30.4?

It is beneficial to run it only when a release branch with bumped minor is created.